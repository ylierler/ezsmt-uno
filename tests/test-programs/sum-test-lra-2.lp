&logic(lra).
&sum{x}<"4.58".
&sum{x}>"4.57".