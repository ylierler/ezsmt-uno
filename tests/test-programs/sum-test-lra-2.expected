{}
x=(/ 1829 400)